extensions/amp-sidebar/1.0/base-element.js

```diff
@@ -40,6 +46,25 @@ export class BaseElement extends PreactBaseElement {
     });
   }
 
+  /** @override */
+  updatePropsForRendering(props) {
+    this.getRealChildNodes().map((child) => {
+      if (
+        child.nodeName === 'NAV' &&
+        child.hasAttribute('toolbar') &&
+        child.hasAttribute('toolbar-target')
+      ) {
+        props['children'].push(
+          <AmpSidebarToolbar
+            toolbar={child.getAttribute('toolbar')}
+            toolbarTarget={child.getAttribute('toolbar-target')}
+            domElement={child}
+          ></AmpSidebarToolbar>
+        );
+      }
+    });
+  }
```

Comment:
Iterate through the amp elements' real children.  For each child that is a `nav` element and has the attributes `toolbar` and `toolbar-target`, add an additional child to be passed to the preact side.

The preact side children currently contains a single `slot`.  This adds one `AmpSidebarToolbar` for each of these nav elements found.